core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -67,6 +67,7 @@ public class JdbcCatalog extends BaseMetastoreCatalog
   private static final String NAMESPACE_EXISTS_PROPERTY = "exists";
   private static final Logger LOG = LoggerFactory.getLogger(JdbcCatalog.class);
   private static final Joiner SLASH = Joiner.on("/");
+  private static final String DEFAULT_FILE_IO_IMPL = "org.apache.iceberg.io.ResolvingFileIO";
```

Comment:
You could use the name from the class potentially, rather than a string literal, as `ResolvingFileIO` is in core.